README.md

```diff
@@ -20,16 +20,16 @@ npm install @kava-labs/javascript-sdk
 
 ### Mainnet
 
-- Chain ID: kava-3
-- REST API endpoint: https://kava3.data.kava.io
+- Chain ID: kava-7
+- REST API endpoint:  https://api.kava.io
 - BEP3 deputy's Kava address: kava1r4v2zdhdalfj2ydazallqvrus9fkphmglhn6u6
```

Comment:
Add Kava deputy addresses for all assets as seen in https://github.com/Kava-Labs/javascript-sdk/pull/55